github-upload-action-secrets

```diff
@@ -63,10 +106,16 @@ def main():
                         help="Only show which secrets would get uploaded where")
     parser.add_argument('-v', '--verbose', action="store_true", default=False,
                         help="Print verbose information")
-    parser.add_argument("secrets_dir", help="directory with one file per secret")
+
+    megr = parser.add_mutually_exclusive_group(required=True)
+    megr.add_argument('--directory',
+                      help="Upload given directory with one file per secret")
+    megr.add_argument('--ssh-keygen', metavar="SECRET_NAME",
+                      help="Generate an SSH key, upload private, public to stdout")
+
     opts = parser.parse_args()
 
-    NAME_RE = r'[a-z][-0-9a-z]*'
+    NAME_RE = r'[a-z][-0-9a-z._]*'
```

Comment:
Rebase error, this should be in the previous commit that introduces the name checking.

Reply:
Fixed.  Thanks for the careful checking.